mne/epochs.py

```diff
@@ -1084,6 +1085,65 @@ def plot_drop_log(self, threshold=0, n_max_plot=20, subject='Unknown',
                              color=color, width=width, ignore=ignore,
                              show=show)
 
+    def plot_image(self, picks=None, sigma=0., vmin=None,
```

Comment:
is this the same docstring for the `plot_epochs_image`, just minus the `epochs` parameter?